Flake8 corrections + Python 3 compatibility

PalNilsson opened the pull request:
* Changed print keywords to Python 3 friendly print() functions
* Renamed file variable to filename, and replaced workdir+filename with os.path.join(workdir, filename)
* Removed harmless whitespace

PalNilsson commented:
Not relevant any more. Closing.

afortiorama commented:
What is not relevant anymore? Sorry I missed the merge request. Will do that though I've corected already the parenthesis. What is the .idea/vcs.xml?

PalNilsson commented:
No I had updated your print usages to Python 3 friendly print() and then corrected one whitespace. You have since removed the prints and corrected the whitespace, so the pull request is not relevant. Ignore the .idea/vcs.xml; it is added by the PyCharm editor.
